scripts/cluster/yarn_cluster_setup/Test_Pi/stop_yarn_spark_cluster.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# root_folder_name=/yarn-spark-cluster
+# HADOOP_HOME=$root_folder_name/hadoop
+# SPARK_HOME=$root_folder_name/spark
```

Comment:
do we still need these?

Reply:
No not really. Those comments have been removed.
The global variables are set for root user, but they cannot be accessed without sudo su. Hence all scripts are directly referencing the path instead of the Global variable.